cmd/server/main.go

```diff
@@ -36,6 +36,7 @@ func main() {
 		addr        = flag.String("listen", "/var/run/kmsplugin/socket.sock", "GRPC listen address")
 		key         = flag.String("key", "", "AWS KMS Key")
 		region      = flag.String("region", "us-east-1", "AWS Region")
+		awsEndpoint = flag.String("aws-endpoint", "", "use this aws endpoint instead of the one generated by aws sdk")
```

Comment:
nit: Can you change this flag to `kms-endpoint`? (since its not _all_ of AWS at one endpoint :) ). Can you also use sentence case and put 'aws' in all-caps (like the other flags do)?

Reply:
I started with that, but then in documentation of struct aws.Config it looked generic enough that i changed it to aws-endpoint :)

i'll change it

Reply:
Since this application only ever talks to KMS, we can use the `Config.Endpoint` field. If it starts to talk to anything else, we'll have to switch to EndpointResolvers

Reply:
make sense. thanks